client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -139,7 +139,10 @@ const TransferPage = ( props: TransferPageProps ) => {
 					mainText={ mainText }
 				/>
 			);
-		} else if ( isEnabled( 'domains/transfer-to-any-user' ) ) {
+		} else if (
+			isEnabled( 'domains/transfer-to-any-user' ) &&
+			! [ 'uk', 'fr', 'ca' ].includes( getTopLevelOfTld( selectedDomainName ) )
```

Comment:
according to https://wordpress.com/support/domains/domain-pricing-and-available-tlds/, we also support `.jp`
```suggestion
			! [ 'uk', 'fr', 'ca', 'ja' ].includes( getTopLevelOfTld( selectedDomainName ) )
```

Reply:
Not sure, I think this is just about extra field data, we could check if anything shows up when you try to purchase one.